README.md

```diff
@@ -94,7 +94,11 @@ hugo new blog/:blog-post.md
 
 Where `:blog-post.md` is the name of the file of your new post.
 
-The theme supports KaTeX which renders math typesetting in markdown document. Simply turn on by `math: true` in your post.
+Optional parameters for your post, added by the theme:
+
+* `math:true` enables math renderig with KaTeX
+* `toc: true` adds a table of contents to your post
```

Comment:
actually it should be `math = true` and `toc = true` in the config (note the equal sign). Could you fix the README (and that typo `renderig`) and also add these two lines, commented out, in the exampleSite's `config.toml`?

Reply:
These parameters should go in a blog post's frontmatter, shouldn't it? In that case it would be correct. But we would need to be more specific in the README as to where this should go.

Reply:
Oh I see, I didn't notice your `or .Params.toc .Site.Params.toc`. Yeah if we can specify in both places then we need to explain it in the readme.

Reply:
Do you think we should allow a user to set `toc` sitewide? I feel like this setting is best on a per-page basis. If a user wants to have every blog post contain table of contents they can easily set that up with an archetype.

Reply:
I think we can give users the option too, but personally I'd use a page's front-matter for this. If so we should remove it from the `.Site.Params` just to be consistent.

Reply:
Yes. @xvallspl let's only check for this parameter in the post's frontmatter. We'll document it thoroughly in the README at the end of the PR.